### DISCUSSION TITLE
Is there a way to get more verbosed information from a hook function?

### DISCUSSION THREAD

**Discussion by tigerinus** [rauc/rauc#1209] [Q&A] [answered]
![image](https://github.com/rauc/rauc/assets/7172560/308fb0e8-785f-4894-a536-0d3c24f89ff7)

I tried `--debug` but not much verbose information has been returned. Is there anything else could be done here?

Thanks.

**Comment by tigerinus** [comment]
nevermind - `exit code 5` tells everything. I was able to trace which step went wrong based on this.

**Reply by ejoerns** [reply, marked as answer]
What you get as output of `rauc install` is just the client log.
You should be able to see output of your hook in the service log.

Try `journalctl -u rauc` if using systemd.
